### PR TITLE
feat: LT-1262 Good Document Sensitivity

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**
https://github.com/uktrade/lite-api/pull/229
https://github.com/uktrade/lite-exporter-frontend/pull/258
https://github.com/uktrade/lite-content/pull/42
https://github.com/uktrade/lite-tests-common/pull/94

### CODE REVIEW

File: ui_automation_tests/features/goods.feature [hunk 1]
```diff
@@ -40,7 +42,18 @@ Feature: I want to edit and remove goods on the goods list
     And I click to manage goods on a standard application
     Then I see there are no goods on the application
     When I click Add a new good
-    And I add a new good with description "New good for application" controlled "Yes" control code "ML8a25b" incorporated "No" and part number "P123"
+    And I add a new good with description "New good for application" controlled "Yes" control code "ML8a25b" incorporated "No" and part number "P123" to the application
     And I enter details for the new good on an application with value "99.98", quantity "13" and unit of measurement "Kilogram(s)" and I click Continue"
     And I attach a document to the good with description "Test good spec 01"
     Then A new good has been added to the application
+
```
Comment (phlptt) [inline ui_automation_tests/features/goods.feature]: extra space

Comment (phlptt) [reply ui_automation_tests/features/goods.feature]: extra line sorry

File: applications/views/goods.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def get(self, request, **kwargs):
 
         filtered_data = []
         for good in goods_list:
-            if good["documents"] and not good["is_good_controlled"] == "unsure":
+            if (good["documents"] or good["missing_document_reason"]) and not good["is_good_controlled"] == "unsure":
```
Comment (stephanieboyd) [inline applications/views/goods.py]: Const for "unsure"

File: goods/forms.py [hunk 3]
```diff
@@ -17,120 +23,139 @@
...
+            TextInput(title=CreateGoodForm.PartNumber.TITLE, name=prefix + "part_number", optional=True),
         ],
         back_link=back_link,
+        default_button_name=CreateGoodForm.BUTTON,
     )
 
     return form
 
 
 def are_you_sure(good_id):
```
Comment (stephanieboyd) [inline goods/forms.py]: Am I sure what?

File: applications/views/goods.py (hunk 2 above)
Comment (BenAndrew-Kainos) [reply applications/views/goods.py]: Good idea. Now in constants.py

File: goods/forms.py (hunk 3 above)
Comment (BenAndrew-Kainos) [reply goods/forms.py]: Terrible name. I've renamed it to `raise_a_clc_query`

**Pull request merged**